src/Components/MovieForm/MovieForm.jsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from 'react';
+import styles from './MovieForm.module.css';
+import { LabeledInput } from './LabeledInput/LabeledInput';
+
+const defaultMovieValues = {
+  title: '',
+  releaseDate: '',
+  imageUrl: '',
+  rating: '',
+  genre: '',
+  runtime: '',
+  description: '',
+};
+
+export const MovieForm = ({ initialValues = defaultMovieValues, onSubmit, genres }) => {
+  const [movie, setMovie] = useState({
+    title: initialValues.title ?? '',
+    releaseDate: initialValues.releaseDate ?? '',
+    imageUrl: initialValues.imageUrl ?? '',
+    rating: initialValues.rating ?? '',
+    genre: initialValues.genre ?? '',
+    runtime: initialValues.runtime ?? '',
+    description: initialValues.description ?? '',
+  });
```

Comment:
Can it be simplified it with something like this?

```suggestion
  const [movie, setMovie] = useState({
    ...defaultMovieValues,
    ...initialValues,
  });
```